Error: undefined constant FileUtils

```bash
In lib/nodejs/src/nodejs/internal.cr:48:7

 48 | FileUtils.mkdir(dir)
      ^--------
Error: undefined constant FileUtils
make: *** [Makefile:2: development] Error 1
```

--- a/src/nodejs/internal.cr
+++ b/src/nodejs/internal.cr
@@ -1,4 +1,5 @@
 require "digest/md5"
+require "file_utils"
 
 module Nodejs::Internal
   extend self

